### PR TITLE
Create contribution instructions

### PR DESCRIPTION
**Pull request by matentzn**
@cthoyt whenever you get a chance, a first rough draft that I think works ok.

### CODE REVIEW

**Review by cmungall — APPROVED** [review]

File: src/docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+Include details about your configuration and environment:
+
+* **Which version of SSSOM toolkit/model are you using?** You can get the exact version by running `sssom --version` in your terminal
+* **What's the name and version of the OS you're using**?
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to SSSOM? You can start by looking through these `beginner` and `help-wanted` issues:
+
+* [Beginner issues][beginner] - issues which should only require a few lines of code, and a test or two.
```
Comment (cthoyt) [inline src/docs/contributing.md:83]: fix links with round parentheses on this and the next line

File: src/docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+2. Follow the [styleguides](#styleguides)
+3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+
+While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+
+## Styleguides
+
+### Git Commit Messages
+
+* Use the present tense ("Add feature" not "Added feature")
```
Comment (cthoyt) [inline src/docs/contributing.md:107]: "imperative mood" is a better description of this

Comment (cthoyt) [reply src/docs/contributing.md:107]: uhh I am a bad reviewer, the next line says this

File: src/docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+
+### Issue and Pull Request Labels
+
+This section lists the labels we use to help us track and manage issues and pull requests. Most labels are used across all mapping commons repositories.
+
+#### Type of Issue and Issue State
+
+| Label name | `mapping-commons/sssom` :mag_right: | `sssom`‑org :mag_right: | Description |
+| --- | --- | --- | --- |
+| `enhancement` | [search][search-sssom-repo-label-enhancement] | [search][search-sssom-org-label-enhancement] | Feature requests. |
```
Comment (cthoyt) [inline src/docs/contributing.md:142]: looks like the links are also bracketed incorrectly here

**Review by cthoyt — APPROVED** [review]
Overall a good start for contribution guidelines. Sometimes it felt like a bit of a maze since there were lots of links from one section of the document to another but if you disregard those and keep reading the order makes sense

File: src/docs/contributing.md (hunk 1 above)
Comment (matentzn) [reply src/docs/contributing.md]: Actually - these are github variables: https://stackoverflow.com/questions/24580042/github-markdown-are-macros-and-variables-possible

File: src/docs/contributing.md (hunk 3 above)
Comment (matentzn) [reply src/docs/contributing.md]: https://stackoverflow.com/questions/24580042/github-markdown-are-macros-and-variables-possible

see above

Comment (cthoyt) [reply src/docs/contributing.md]: Wow that's so cool! Thank you for linking me.

**Pull request merged**